### DISCUSSION TITLE
Live table overflow

### DISCUSSION THREAD

**Discussion by mikeassel** [Textualize/rich#2430] [Q&A]
When using `Live` to render a large table, I have set `vertical_overflow='visible'`, but the behavior is a bit odd as the header seems to be printed every time the table is updated. Am I doing something wrong or is there a way to handle this?

**Comment by willmcgugan** [comment]
Can you post a video of what you're seeing.

**Reply by mikeassel** [reply]
I was initially testing in the terminal from within VScode, but just tried it in iTerm and the behavior is different. Will post a video soon. Thanks.

**Reply by mikeassel** [reply]
Here's what I'm seeing
![no_scroll](https://user-images.githubusercontent.com/16667708/182157147-f25b8b17-7996-49a6-9098-113d87905cbf.gif)

**Reply by willmcgugan** [reply]
Your table is too large to fit in the screen. There's not much Rich can do about that, unless you enlarge your terminal manually.

You might want to consider a scrollbar for this, or looking to building a TUI, possibly with Textual.

**Reply by mikeassel** [reply]
OK. Does rich support a scroll bar? I searched the docs but didn't find anything.

**Reply by mikeassel** [reply]
I've been meaning to check out Textual, so this is good motivation :)

**Comment by wardmw** [comment]
> You might want to consider a scrollbar for this, or looking to building a TUI, possibly with Textual.

Does the rich module include any kind of scrollbar? I cannot find it anywhere in the docs. 

If not, I am happy to create a request for it.

**Reply by willmcgugan** [reply]
No, and it's not likely to be added. It's beyond the remit of Rich. If you want scrolling there is [Textual](https://github.com/Textualize/textual) which is built on Rich.

**Reply by wardmw** [reply]
Thanks, I will take a look.
